xfer/resolver.go

```diff
@@ -1,4 +1,4 @@
-package main
+package xfer
```

Comment:
The same could be said for {Multi,Background,HTTP}Publishers and (Multi)AppClient...  The resolver should very much be colocated this there 5 things; I'm open to arguments that these should be in a probe/ sub module...